src/datasets.py

```diff
@@ -1,10 +1,11 @@
 import copy
-from typing import Tuple
+from typing import Tuple, Union
 
 import torch_geometric.datasets as datasets
 import torch_geometric.transforms as T
 from torch.utils.data import random_split
 from torch_geometric.data import Dataset
+from torch_geometric.loader import DataLoader
 from torch_geometric.transforms import base_transform
 
 _DATASET_NAMES = dict(
```

Comment:
Minor suggestion: perhaps rename `get_dataset` as `get_datasets` to (1) be consistent with `get_dataloaders` and (2) suggest to caller that it'll return more than one dataset objects? (Again, sorry I couldn't figure out how to comment to the correct line.)